src/routes/ChatRoom/ChatRoom.js

```diff
@@ -371,6 +375,34 @@ export default function ChatRoom (props) {
     }, [])
   }, [postsPast, postsFuture])
 
+  useEffect(() => {
```

Comment:
Hmm. Seems like there should be a way to load the posts starting at the postid instead of loading and then scrolling to a new post